USAGE.md

```diff
@@ -52,11 +52,12 @@ kubectl patch deployment -n ingress-nginx ingress-nginx-controller --type='json'
 ```
 
 ## 5. Deploy Vault
-Run the following commands to deploy Vault:
+Download Atmos, follow the [instructions](https://atmos.tools/quick-start/install-atmos) upstream.
+
+Run the following commands to deploy the component vault-deployment to the playground stack:
 ```bash
-cd Vault-Deployment
-terraform init
-terraform apply
+atmos terraform plan vault-deployment -s playground
```

Comment:
- basically the very same commands, just prepend the atmos magic and tell atmos which stack (-s playground) to provision with what component ("vault-deployment")
- we could think about building a minikube and a kind stack